accesscontrol_usermanagement/configuring-sso-for-argo-cd-using-keycloak.adoc

```diff
@@ -23,9 +23,17 @@ include::modules/gitops-logging-into-keycloak.adoc[leveloffset=+1]
 // Uninstalling Keycloak
 include::modules/gitops-uninstall-keycloak.adoc[leveloffset=+1]
 
+// Modifying Red Hat build of Keycloak resource requests/limits
```

Comment:
```suggestion
// Modifying RHSSO resource requests/limits
```
We have not yet switched to Red Hat build of Keycloak versions for SSO so lets use RHSSO term everywhere.

Reply:
Fixed.